src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -22,6 +22,7 @@
         "symfony/class-loader": "~3.2",
         "symfony/dependency-injection": "^3.4.24|^4.2.5",
         "symfony/config": "~3.4|~4.0",
+        "symfony/debug": "~2.8|~3.0|~4.0",
```

Comment:
`FrameworkBundle` registers `Symfony\Component\Debug\ErrorHandler` when `boot()` is called